README.md

```diff
@@ -91,10 +91,12 @@ The easiest way to run releasetool is via docker.
 2. Copy `zaptain_ui/settings_template.py` to `zaptain_ui/settings_local.py` and adjust the File to your needs.
 Especially change the `SECRET_KEY` variable and remove the line below it.
 One possibility to create a secret is `< /dev/urandom tr -dc _A-Z-a-z-0-9 | head -c${1:-64};echo;`.
+The variable `MEDIA_ROOT` contains the location of the database file. and `$MEDIA_ROOT/releasetool` will contain the releasecandidates.
```

Comment:
The database file is defined in the ``DATABASES`` setting. The database file should be placed somewhere in ``/rt_data`` (because this folder is created in the Dockerfile for that purpose). E.g.
```
DATABASES = {
    'default': {
        'ENGINE': 'django.db.backends.sqlite3',
        'NAME': '/rt_data/rt_v1alpha.sqlite',
    }
}
```
The release candidates are placed in ``$MEDIA_ROOT/releasecandidates``. ``MEDIA_ROOT`` should be a relative path to ``/releasetool`` , e.g. ``MEDIA_ROOT=rt_data`` .